packages/create/bin/create-gene-workspace.js

```diff
@@ -96,6 +96,7 @@ async function runCommands() {
       'brainly-style-guide',
       `inversify@5.1.1`,
       `@tanstack/react-query@5.55.4`,
+      '@nrwl/next@15.8.6',
```

Comment:
it is also dev dep

Reply:
that's correct way of using it, btw https://nx.dev/recipes/next/next-config-setup#composing-plugins-using-composeplugins-utility-nx-16-and-later